src/main/java/com/naveenautomation/Pages/ShoppingCartPage.java

```diff
@@ -21,6 +35,42 @@ protected String getPageUrl() {
 		return getDomain() + PAGE_URL;
 	}
 
+	private void selectCountryFromDropDown(String countryValue) {
+		((ProxyDriver) wd).selectFromDropDown(((ProxyDriver) wd).findElement(countryDropDown), countryValue);
+	}
+
+	private void selectRegionFromDropDown(String regionValue) {
+		((ProxyDriver) wd).selectFromDropDown(((ProxyDriver) wd).findElement(regionDropDown), regionValue);
+	}
+
+	private void inputPostalCode(String pinCode) {
+		((ProxyDriver) wd).sendKeys(postalCodeInputBox, pinCode);
+	}
+
+	private void clickGetQuotesBtn() {
+		((ProxyDriver) wd).click(getQuotesBtn);
```

Comment:
is this btn opening a page, or dialog, if yes then it should not be void

Reply:
Further to our discussion, I have created a new page for modal popup window.